src/App.js

```diff
@@ -1,24 +1,129 @@
+import { useState, useEffect } from 'react';
 import './App.css';
+import './styles.css';
+
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <>
+    <NavBar></NavBar>
+
+    <div className="friendstop">
+      <ul>
+        <h1>FRIENDS</h1>
+        <input type="text" placeholder="Search by username..."></input>
+      </ul>
+    </div>
+      <div className="friendrequestoutercontainer">
+        <div className="friendrequestinnercontainer">
+        <h1>Friend Requests</h1>
+          <ul>
+            <FriendRequest name="Konrad Radecki" username="the1krad_" />
+            <FriendRequest name="Finn McMillan" username="finnmcm_" />
+          </ul>
+        </div>
+      </div>
+      
+      
+  <div className = "friendcardcontainer">
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+    <FriendCard></FriendCard>
+  </div>
+    </>
+  );
+}
+
+
+function NavBar() {
+  return (
+    <>
+    <nav className="navbar">
+      <ul>
+        <input className="searchbar" type="text" placeholder="Search users..."></input>
+        <li><a href="">pfpdropdown</a></li>
+        <li><a href="">Friends</a></li>
+        <li><a href="index.html">status.NU</a></li>
+      </ul>
+      </nav>
+    </>
+  );
+}
+
+
+function FriendCard() {
+  return (
+    <>
+
+      <div className='friendcard'>
+        <ul>
+          <h2>Konrad</h2>
+          <h4>MATH 228-2 in 15m</h4>
+        </ul>
     </div>
+    </>
+  );
+}
+
+
+function FriendRequest (props) {
+
+  const [decisionmessage, setDecisionmessage] = useState ('');
+  const [friendstatus, setFriendstatus] = useState ('pending')
+  const [showList, setShowList] = useState (true);
+
+
+
+  const handleClickAccept = () => {
+    setDecisionmessage('Friend request accepted!');
+    setFriendstatus('accepted')
+    setShowList(false)
+    console.log(friendstatus)
+
+    }
+  
+  const handleClickDeny = () => {
+    setDecisionmessage('Friend request denied!')
+    setFriendstatus('denied')
+    setShowList(false)
+    console.log(friendstatus)
+
+
+  }
+
```

Comment:
You haven't used a `useEffect` in your code. It could be added  in the `FriendRequest` component to watch for changes to the `friendstatus` state....This would log a message to the console whenever a friend requests status is updated to either "accepted" or "denied."